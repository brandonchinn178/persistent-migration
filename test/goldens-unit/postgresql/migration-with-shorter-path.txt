CREATE TABLE IF NOT EXISTS "person" ( "gender" VARCHAR )
