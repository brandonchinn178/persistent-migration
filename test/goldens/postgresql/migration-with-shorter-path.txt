CREATE TABLE "person"("gender" VARCHAR)
